BoxContentSDK/BoxContentSDK/Models/BOXItem.m

```diff
@@ -225,7 +225,14 @@ - (instancetype)initWithJSON:(NSDictionary *)JSONResponse
                                                                   inDictionary:JSONResponse
                                                                hasExpectedType:[NSArray class]
                                                                    nullAllowed:NO];
-        
+        // Parse DefaultInviteeRole
```

Comment:
super minor: style/readability, might want to allow a line between the allowed roles and default role pieces.